.gitignore

```diff
@@ -8,4 +8,4 @@ wasi-testsuite/
 dependency-reduced-pom.xml
 jmh-result.json
 scripts/jmh-tmp
-main/
+/main/
```

Comment:
@electrum I understand now the motivation of your change in #288 I think this is good enough for now.

Reply:
Yep, this is good, thanks